src/components/SchemaResourcePage/SchemaResourceEdit.tsx

```diff
@@ -143,7 +161,12 @@ export function SchemaResourceEdit({
     });
   }, [register, formFields, watch]);
 
-  const doSubmit = (props: any) => {
+  const doSubmit = async (props: any) => {
+    const specErrorMessage = await schemaSpecValidator(props?.["spec"]);
+    if (specErrorMessage) {
+      console.error(specErrorMessage);
```

Comment:
These error messages should be shown on the form / modal

Reply:
Done @moshloop